commands/utils/set_args.js

```diff
@@ -385,6 +386,36 @@ function sync_args_from_cmd(args) {
         }
       }
     }
+    let dot_env_vars = undefined;
+    if ("sys-env-keys" in args) {
+      dot_env_vars = args["sys-env-keys"];
+    } else if (lt_config["run_settings"]["sys_env_keys"]) {
```

Comment:
can we check it like lt_config["run_settings"] && lt_config["run_settings"]["sys_env_keys"]

Reply:
done